cms/core/templates/blocks/acf/visit_nhsuk_infobar_block.html

```diff
@@ -1,4 +1,3 @@
-<div class="{{ classname }}">
-  <p></p>Visit nhs.uk
-  Find medical advice at nhs.uk</p>
+<div class="nhs-banner">
+  <span><strong>For any medical advice relating to mental health please visit <a href="#">NHS.uk</a></strong></span>
```

Comment:
to backend person: "For any medical advice please visit" if no string, otherwise "For any medical advice relating to {{thing}} please visit.